### DISCUSSION TITLE
Welcome to QA Discussions!

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#447] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪

**Comment by pattacini** [comment]
Hey @robotology/everyone 👋🏻 

We've just enabled the brand new GitHub feature for hosting "[Discussions](../discussions)" ✨
Discussions is a more robust and proper way of supporting our community than the present issue system.

To learn more, please refer to the official [GitHub Documentation](https://docs.github.com/en/free-pro-team@latest/discussions).

Therefore, in the upcoming weeks, we will be phasing out issues in favor of discussions.

Don't worry, this process will take some time to complete and, although you can already open discussions for querying a question, you can still rely on issues. 

As part of the migration, we will also make sure to convert old/closed issues to corresponding discussions, while being careful in assigning the correct category.

**Reply by pattacini** [reply]
Apologies in advance for the "noise" we are going to create while moving issues into discussions items 🙏🏻

Apparently, bulk migration can be applied only to open issues; thus, our old closed issues need to be opened again generating an annoying burst of notifications.

Migrating old issues into discussions will preserve our repertoire of knowledge.

**Reply by drdanz** [reply]
One thing that I don't like so far, is that it is not possible to assign labels to the discussions, just a category.
For example I would like to add a "YARP" label to #132, but it doesn't seem to be possible.

**Comment by pattacini** [comment]
@robotology/everyone to prevent GitHub from bothering you with the bunch of notifications I'll certainly generate during the transitions in the upcoming days, you may consider customizing your subscription to this repository in order to select only Discussions and thus silencing notifications related to issues.

Here's how you can get this done: https://github.blog/2020-11-20-whatcha-watchin-custom-notification-controls.

**Reply by pattacini** [reply]
Hello @robotology/everyone 

This is to let you know that we've finally completed the migration of all QA/issues into QA/discussions ✔
Certain newly ported Q&A discussions still need to be revised by us in order to identify the best answers but this won't prevent you from perusing their content. 

As a result, we have just **disabled the issues system to fully favor discussions** from now onward.
Have a glance at the [README][1] for the updated instructions.

[1]: ../blob/master/README.md
